addons/mass_mailing/models/mass_mailing.py

```diff
@@ -478,7 +478,7 @@ def read_group(self, cr, uid, domain, fields, groupby, offset=0, limit=None, con
         if groupby and groupby[0] == "state":
             # Default result structure
             # states = self._get_state_list(cr, uid, context=context)
-            states = [('draft', 'Draft'), ('test', 'Tested'), ('done', 'Sent')]
+            states = [('draft', _('Draft')), ('test', _('Tested')), ('done', _('Sent'))]
```

Comment:
As a result of the previous comment, this change is incorrect as well because selection labels are using a different type of translation than what is handled by the source code translation function (`_()`).
A simple alternative is to use the already-translated list of labels from the `state` field definition, instead of redefining them. Something like this could do the trick:
```python
  states = dict(self.fields_get(cr, uid, ['state'], context=context)['state']['selection'])
```